src/link/MachO/zld.zig

```diff
@@ -338,6 +351,20 @@ pub fn linkWithZld(
             }
 
             Compilation.dump_argv(argv.items);
+
+            print: {
+                std.debug.getStderrMutex().lock();
+                defer std.debug.getStderrMutex().unlock();
+                const stderr = std.io.getStdErr().writer();
+                if (options.lib_dirs.len != 0) {
+                    nosuspend stderr.print("Library search paths:\n", .{}) catch break :print;
+                    for (options.lib_dirs) |dir| nosuspend stderr.print("\t{s}\n", .{dir}) catch break :print;
+                }
+                if (options.framework_dirs.len != 0) {
+                    nosuspend stderr.print("Framework search paths:\n", .{}) catch break :print;
+                    for (options.framework_dirs) |dir| nosuspend stderr.print("\t{s}\n", .{dir}) catch break :print;
+                }
+            }
```

Comment:
Should we not try moving this into the frontend where we resolve those paths and have it available under `--verbose` flag?

Reply:
I agree, will do in a follow-up commit.

Reply:
For now, I will remove this printing from PR.

In moving to the front end, things got messy real fast because we have code all over the place (frontend / compilation / linkers) deciding on search dirs. I wasn't happy with _not_ printing the final SDK/vendored dirs under certain combinations of command line switches, this led to that rabbit hole.